nvflare/private/fed/server/server_runner.py

```diff
@@ -201,77 +201,45 @@ def process_task_request(self, client: Client, fl_ctx: FLContext) -> (str, str,
         if not isinstance(engine, ServerEngineSpec):
             raise TypeError("engine must be ServerEngineSpec but got {}".format(type(engine)))
 
-        self.log_info(fl_ctx, "got task request from client")
+        self.log_debug(fl_ctx, "process task request from client")
 
         if self.status == "init":
-            self.log_info(fl_ctx, "server runner still initializing - asked client to try again later")
+            self.log_error(fl_ctx, "server runner still initializing - asked client to try again later")
             return self._task_try_again()
 
         if self.status == "done":
-            self.log_info(fl_ctx, "server runner is finalizing - asked client to end the run")
+            self.log_error(fl_ctx, "server runner is finalizing - asked client to end the run")
             return SpecialTaskName.END_RUN, "", None
 
         peer_ctx = fl_ctx.get_peer_context()
         if not isinstance(peer_ctx, FLContext):
-            self.log_error(fl_ctx, "invalid task request: no peer context, asked client to try again.")
+            self.log_error(fl_ctx, "invalid task request: no peer context - asked client to try again later")
             return self._task_try_again()
 
         peer_run_num = peer_ctx.get_run_number()
         if not peer_run_num or peer_run_num != self.run_num:
-            # the client is on a different RUN
-            self.log_info(fl_ctx, "invalid task request: not the same run_number, asked client to end run")
+            # the client is in a different RUN
+            self.log_error(fl_ctx, "invalid task request: not the same run_number - asked client to end the run")
             return SpecialTaskName.END_RUN, "", None
 
         try:
             with self.wf_lock:
                 if self.current_wf is None:
-                    self.log_info(fl_ctx, "There's no current workflow - asked client to try again later")
+                    self.log_error(fl_ctx, "no current workflow - asked client to try again later")
                     return self._task_try_again()
 
                 task_name, task_id, task_data = self.current_wf.responder.process_task_request(client, fl_ctx)
 
                 if not task_name or task_name == SpecialTaskName.TRY_AGAIN:
-                    self.log_info(fl_ctx, "no task currently for client - asked client to try again later")
+                    self.log_error(fl_ctx, "no task currently for client - asked client to try again later")
```

Comment:
this will always show under "try_again", I think we can move it to debug

Reply:
Yes, debug log level makes sense to me.